src/test/.eslintrc.json

```diff
@@ -1,6 +1,9 @@
 {
   "env": {
-      "jest": true
-    }
+    "jest": true
+  },
+  "rules": {
+    "react/jsx-no-bind": 0
```

Comment:
we use an arrow function as a prop in a test, but we don't really care because it's tests